packages/apputils-extension/src/index.ts

```diff
@@ -195,6 +195,13 @@ const themes: JupyterLabPlugin<IThemeManager> = {
       currentTheme = args.newValue;
       app.shell.dataset.themeLight = String(manager.isLight(currentTheme));
```

Comment:
Is this attribute not sufficient for what you need? The intention here was to give a CSS selector that allows you to use different icon classes depending on whether it is light or dark.